Sources/Load.php

```diff
@@ -1739,7 +1741,7 @@ function loadMemberContext($user, $display_custom_fields = false)
 		if (!empty($image))
 			$memberContext[$user]['avatar'] = array(
 				'name' => $profile['avatar'],
-				'image' => '<img class="avatar" src="' . $image . '" alt="">',
+				'image' => '<img class="avatar" src="' . $image . '" alt="" loading="lazy" width="' . $profile['attachment_width'] . 'px" height = "'. $profile['attachment_height'] . 'px">',
```

Comment:
> The width and height attributes on img, iframe, embed, object, video, and, when their type attribute is in the Image Button state, input elements may be specified to give the dimensions of the visual content of the element (the width and height respectively, relative to the nominal direction of the output medium), in CSS pixels. The attributes, if specified, must have values that are valid **non-negative integers**.
https://html.spec.whatwg.org/multipage/embedded-content-other.html#attr-dim-width

So, you should use `width="' . $profile['attachment_width'] . '" height = "'. $profile['attachment_height'] . '"`, only numbers, without `px`.